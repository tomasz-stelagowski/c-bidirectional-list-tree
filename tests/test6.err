NODES: 2
NODES: 3
NODES: 4
NODES: 5
NODES: 6
NODES: 7
NODES: 8
NODES: 9
NODES: 10
NODES: 11
NODES: 12
NODES: 13
NODES: 14
NODES: 15
NODES: 16
NODES: 15
NODES: 16
NODES: 15
NODES: 15
NODES: 15
NODES: 15
NODES: 16
NODES: 15
NODES: 16
NODES: 13
NODES: 14
NODES: 15
NODES: 15
NODES: 16
NODES: 15
NODES: 16
NODES: 15
NODES: 15
NODES: 15
NODES: 15
NODES: 16
NODES: 15
NODES: 16
NODES: 15
NODES: 15
NODES: 15
NODES: 15
NODES: 16
NODES: 15
NODES: 16
NODES: 15
NODES: 15
NODES: 15
NODES: 15
NODES: 16
NODES: 15
NODES: 16
NODES: 15
NODES: 15
NODES: 15
NODES: 15
NODES: 16
NODES: 15
NODES: 16
NODES: 14
NODES: 14
NODES: 14
NODES: 14
NODES: 14
NODES: 14
NODES: 14
