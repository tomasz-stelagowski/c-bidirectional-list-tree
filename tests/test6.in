ADD_NODE 0
SPLIT_NODE 0 1
ADD_NODE 2
SPLIT_NODE 0 2
ADD_NODE 2
SPLIT_NODE 2 5
ADD_NODE 5
SPLIT_NODE 5 7
ADD_NODE 4
SPLIT_NODE 4 9
ADD_NODE 6
SPLIT_NODE 4 9
ADD_NODE 6
SPLIT_NODE 2 6
ADD_NODE 11
DELETE_NODE 10
SPLIT_NODE 2 6
DELETE_SUBTREE 8
RIGHTMOST_CHILD 2
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 14
ADD_NODE 0
DELETE_NODE 15
SPLIT_NODE 0 17
DELETE_SUBTREE 4
SPLIT_NODE 0 17
ADD_NODE 18
RIGHTMOST_CHILD 19
ADD_NODE 17
DELETE_NODE 6
SPLIT_NODE 2 16
DELETE_SUBTREE 21
RIGHTMOST_CHILD 22
RIGHTMOST_CHILD 5
RIGHTMOST_CHILD 14
ADD_NODE 7
DELETE_NODE 2
SPLIT_NODE 0 13
DELETE_SUBTREE 22
RIGHTMOST_CHILD 14
RIGHTMOST_CHILD 3
RIGHTMOST_CHILD 7
ADD_NODE 16
DELETE_NODE 1
SPLIT_NODE 0 13
DELETE_SUBTREE 13
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 14
RIGHTMOST_CHILD 24
ADD_NODE 0
DELETE_NODE 16
SPLIT_NODE 26 24
DELETE_SUBTREE 25
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 26
RIGHTMOST_CHILD 27
ADD_NODE 28
DELETE_NODE 23
SPLIT_NODE 24 17
DELETE_SUBTREE 5
RIGHTMOST_CHILD 17
RIGHTMOST_CHILD 26
RIGHTMOST_CHILD 14
RIGHTMOST_CHILD 3
RIGHTMOST_CHILD 24
RIGHTMOST_CHILD 30
