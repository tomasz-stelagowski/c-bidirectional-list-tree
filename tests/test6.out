OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
16
4
-1
OK
OK
OK
OK
OK
OK
18
OK
OK
OK
OK
-1
7
-1
OK
OK
OK
OK
-1
-1
23
OK
OK
OK
OK
26
-1
19
OK
OK
OK
OK
27
28
-1
OK
OK
OK
OK
-1
28
-1
-1
30
19
